presto-main/src/main/java/com/facebook/presto/sql/analyzer/MaterializedViewQueryOptimizer.java

```diff
@@ -251,10 +280,15 @@ protected Node visitSelect(Select node, Void context)
     @Override
     protected Node visitSingleColumn(SingleColumn node, Void context)
```

Comment:
Can we add a case where it is grouped by a function call based alias

Reply:
Could you give me an example of query grouped by a function call?

Reply:
nvm you already answered above.